src/contexts/ActivityContext.js

```diff
@@ -27,26 +27,29 @@ const SymbolsByName = new Map(
   })
 )
 
-const getStoredList = (daoDomain, account) =>
-  new StoredList(`activity:${network.type}:${daoDomain}:${account}`, {
-    preStringify: activity => ({
-      ...activity,
-      status: activity.status.description.replace('ACTIVITY_STATUS_', ''),
-      type: activity.type.description.replace('ACTIVITY_TYPE_', ''),
-    }),
-    postParse: activity => ({
-      ...activity,
-      status: SymbolsByName.get(`ACTIVITY_STATUS_${activity.status}`),
-      type: SymbolsByName.get(`ACTIVITY_TYPE_${activity.type}`),
-    }),
-  })
+const getStoredList = (daoAddress, account) =>
+  new StoredList(
+    `activity:chainId-${network.chainId}:${daoAddress}:${account}`,
```

Comment:
Note that this key change will reset any saved activities a user may have held.

In particular, I found it better to use an address here rather than the domain—the same organization can hold multiple (or no) ENS domains.